app/controllers/bookings_controller.rb

```diff
@@ -32,6 +32,7 @@ def update
 
   def destroy
     @booking.destroy
+    redirect_to bookings_path
```

Comment:
You need to do Booking.find(params[:id]) first otherwise @booking will be nil I think!